timescaledb/overview/how-does-it-compare/timescaledb-vs-postgres.md

```diff
@@ -1,251 +1,197 @@
-complex planning (given that a single hypertable can be comprised of
-many chunks).  This can translate to a few extra milliseconds of
-planning time, which can have a disproportional influence for very
-low-latency queries (< 10&nbsp;ms).
+## Lower storage costs
 
-## Time-oriented features
+With compression and downsampling, TimescaleDB can dramatically reduce the size
+of your tables and reduce your storage costs.
 
-TimescaleDB also includes a number of time-oriented features that
-aren't found in traditional relational databases. These include
-special query optimizations (like the merge append above) that provide
-some of the huge performance improvements for time-oriented queries,
-as well as other time-oriented functions (some of which are listed below).
+### Greater than 90% storage savings with best-in-class compression algorithms
 
-#### Time-oriented analytics
+TimescaleDB provides [native columnar compression][compression]. Users often see
+their disk consumption decrease by over 90%, compared to storing the same amount
+of data in standard PostgreSQL. If you're using [Timescale Cloud][cloud], which
+decouples billing for compute and storage, enabling compression significantly
+decreases your storage bill.
 
-TimescaleDB includes _new_ functions for time-oriented analytics,
-including some of the following:
+This chart shows the size of an example dataset when stored in TimescaleDB with
+compression, compared to its size in a regular PostgreSQL database. For more
+information on the comparison, see the [comparison blog
+post][blog-postgresql-vs-timescaledb].
+
+<img class="main-content__illustration"
+src="https://www.timescale.com/blog/content/images/2022/09/image-5.png"
+alt="Storage size of a dataset in TimescaleDB compared to PostgreSQL.
+TimescaleDB stores the data in 8.6 GB, while standard PostgreSQL stores the data
+in 159 GB." />
+
+With compression policies,  chunks can be compressed automatically once all data
+in the chunk has aged beyond the specified time interval. In practice, this
+means that a hypertable can store data as row-oriented for newer data and
+column-oriented for older data.
+
+Having the data stored as both row and column store also matches the typical
+query patterns of time-series applications. This improves overall query
+performance.
+
+### Automatic downsampling and removal of old data with one command
+
+To save even more on storage costs, you can set up an automatic [data retention
+policy][retention] with one SQL command. To set this up in standard PostgreSQL,
+you'd either need to `DELETE` individual records, which is an inefficient
+operation, or set up declarative partitioning and automation yourself.
+
+In TimescaleDB, you can combine continuous aggregates and data retention
+policies to downsample data, and then drop the raw measurements. This allows you
+to retain higher-level rollups of historical data. You have control over the
+granularity of your data and your storage costs.
```

Comment:
I wonder if we can present the Timescale case here first, then compare it to the PG case?